docker-compose.yml

```
@@ -0,0 +1,41 @@
+    ports:
+      - 9090:9090
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+
+  redis-exporter:
+    image: oliver006/redis_exporter:latest
+    command:
+  #    - '--redis.addr=host.docker.internal:6379'
+      - '--redis.addr=127.0.0.1:6379'      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)